face_recognition:
  ros__parameters:
      input: rpi
      local_gallery_file: local_gallery.json
      similarity_threshhold: 0.35
      video_width: 1280
      video_height: 720
      video_fps: 15